projects/ion/src/lib/smart-table/smart-table.component.spec.ts

```diff
@@ -616,4 +616,9 @@ describe('Table without Data', () => {
     await sut(tableWithoutData);
     expect(screen.getByText('Não há dados')).toBeInTheDocument();
   });
+
+  it('should the checkbox be disabled, when there is no data', async () => {
```

Comment:
```suggestion
  it('checkbox should be disabled when there is no data', async () => {
```
Is not a problem don't starting these sentences with "should".